components/atom/textarea/src/index.scss

```diff
@@ -4,7 +4,10 @@
 $short-num-lines: 5 !default;
 $long-num-lines: 7 !default;
 
-$pm-atom-textarea: $p-m !default;
+$p-atom-textarea: 2px !default;
+$pm-atom-textarea: $p-m !default; // deprecated
+$pt-atom-textarea: $p-m !default;
```

Comment:
`$pt-atom-textarea` and `$pl-atom-textarea` default value should be `$pm-atom-textarea` (the deprecated one). As if it was used before, we should keep the value used.

Reply:
Done. there was an error on `pl-whatever` i actually update